experimental/ixdtf/src/parser.rs

```diff
@@ -69,6 +73,7 @@ pub struct ParsedDateTime {
     pub minute: Option<u8>,
     pub second: Option<u8>,
     pub nano_second: Option<i32>,
+    pub time_zone: Option<Vec<u8>>,
```

Comment:
Want to support `Option<[u8]>` but don't know how to do it.

Reply:
you need to add a lifetime to the struct, and then the type becomes `Option<&'a [u8]>`

Reply:
sg!